specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The store ID - must use the tenant ID"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the request."
+    },
+    "OfferIdPathParameter": {
+      "in": "path",
+      "name": "OfferId",
+      "type": "string",
+      "required": true,
+      "description": "The offer ID to update or delete"
+    },
+    "offerObjectParameter": {
+      "in": "body",
+      "name": "offerObject",
+      "schema": {
+        "$ref": "#/definitions/offerSkuObject"
+      }
+    },
+    "offerSkuObjectParameter": {
+      "in": "body",
+      "name": "skuObject",
+      "schema": {
+        "$ref": "#/definitions/offerSkuObject"
+      }
+    }
+  },
+  "responses": {
+    "PrivateStoreAddOffer422": {
+      "description": "Offer with given id already exists in the collection"
+    },
+    "PrivateStoreOffer404": {
+      "description": "The offer id is could not be found in this private store collection"
+    }
+  },
+  "definitions": {
+    "offerSkuObject": {
+      "type": "object",
+      "required": [
+        "isAvailable"
+      ],
+      "properties": {
+        "isAvailable": {
+          "type": "boolean"
+        }
+      }
+    },
+    "offerObject": {
+      "type": "object",
+      "required": [
+        "isAvailable"
+      ],
+      "properties": {
+        "isAvailable": {
+          "type": "boolean"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates Microsoft.Marketplace service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "properties": {
+            "code": {
+              "description": "Error code.",
+              "type": "string",
+              "readOnly": true
+            },
+            "message": {
+              "description": "Error message indicating why the operation failed.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "PrivateStoreList": {
+      "description": "Describes the json payload for the list of available private stores (between zero and one, inclusive)",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateStoreProperties"
+          }
+        }
+      }
+    },
+    "PrivateStoreProperties": {
+      "description": "Describes the json payload on whether or not the private store is enabled for a given tenant",
+      "type": "object",
+      "properties": {
+        "availability": {
+          "type": "string",
+          "enum": [
+            "enabled",
```

Comment:
model enums as strings  (search repo for modelAsString)

Reply:
done